typewise-alert.c

```diff
@@ -1,71 +1,79 @@
-#include "typewise-alert.h"
 #include <stdio.h>
+#include "typewise-alert.h"
+#include"checker.h"
+
+BreachType inferBreach(double value, double lowerLimit, double upperLimit)
+{
+	BreachType breach = NORMAL;
+	checkLowLimitBreach(value, lowerLimit, &breach);
+	checkUpLimitBreach(value, upperLimit, &breach);
```

Comment:
This code can be optimized. If the low level breach occurs then checking high level can be avoided or vice versa.